src/Raven.Client/Documents/Session/IAsyncDocumentSession.TimeSeriesFor.cs

```diff
@@ -8,19 +8,72 @@
 namespace Raven.Client.Documents.Session
 {
     /// <summary>
-    /// Provides an access to DocumentSession TimeSeries API.
+    /// Provides an access to DocumentSession Time Series API.
     /// </summary>
     public partial interface IAsyncDocumentSession
     {
+        /// <summary>
+        /// Provides access to the Time Series API for performing server-side operations such as Append, Get, and Delete on a specific time series.
+        /// </summary>
+        /// <param name="documentId">The unique identifier of the document associated with the time series.</param>
+        /// <param name="name">The name of the time series to access.</param>
+        /// <inheritdoc cref="DocumentationUrls.Session.TimeSeries.ClientApi"/>
         IAsyncSessionDocumentTimeSeries TimeSeriesFor(string documentId, string name);
+
+        /// <inheritdoc cref="TimeSeriesFor"/>
+        /// <param name="entity">The document object associated with the time series.</param>
+        /// <param name="name">The name of the time series to access.</param>
         IAsyncSessionDocumentTimeSeries TimeSeriesFor(object entity, string name);
+
+        /// <inheritdoc cref="TimeSeriesFor"/>
+        /// <typeparam name="TValues">The type of values stored in the time series.<br/>
+        /// See <see href="https://ravendb.net/docs/article-page/6.0/csharp/document-extensions/timeseries/client-api/named-time-series-values"/>.</typeparam>
```

Comment:
This should be referenced from `DocumentationUrls.cs`

Reply:
done